cmd/fpga_crihook/main.go

```diff
@@ -189,6 +189,16 @@ func (he *hookEnv) programBitStream(params *fpgaParams, fpgaBitStreamPath string
 	if err != nil {
 		return fmt.Errorf("failed to program AFU %s to region %s: error: %v, output: %s", params.afu, params.region, err, string(output))
 	}
+
+	programmedAfu, err := he.getProgrammedAfu(params.devNum)
+	if err != nil {
+		return err
+	}
+
+	if programmedAfu != params.afu {
```

Comment:
are we sure here that params.afu is always in the same format ? e.g. can't it be for some reason in mixed case ?